.devcontainer/Dockerfile

```diff
@@ -153,6 +153,17 @@ RUN git clone --recurse-submodules -b v1.35.0 https://github.com/grpc/grpc && \
     cd / && \
     rm -rf grpc
 
+##### libprotobuf-mutator is used for randomized proto unit tests / property tests
+RUN git clone -b v1.0 https://github.com/google/libprotobuf-mutator && \
+    cd libprotobuf-mutator && \
```

Comment:
you can mkdir directly into `libprotobuf-mutator` + cd into it. Save you one line